### ISSUE TITLE
Package Request: Virtualbox

### ISSUE THREAD

**Issue by connor557** [solus-project/repository#106]
I would like to have virtualbox so that I can run other os's inside my Solus. I do a lot of visualization work and it would be nice to be able to test operating systems before I deploy them.

**Comment by JoshStrobl** [comment]
Not really a reason to have Virtualbox in the repository when Qemu already is (an arguably superior virtualizer / machine emulator).

**Comment by funnylookinhat** [comment]
I think there's an argument for VirtualBox - people are more familiar with it and there are quite a few popular tools that require it ( such as IEVMS https://github.com/xdissent/ievms ).

In terms of priority though - hard to say... it would require dkms to be built into the kernel for most of it's usefulness...  so that's kind of a pain.

**Comment by connor557** [comment]
I really prefer Virtualbox over QEMU. Also, I do a lot of docker testing which needs dkms/virtualbox as well.

**Comment by b1be** [comment]
Not really a bug. Virtual box runs fine when compiled from source. I ran WindowsXP under VBox on SolusOS. Stop Requesting things that already works.

**Comment by JoshStrobl** [comment]
@b1be 

Umm excuse me? Just because it is buildable from source doesn't mean there shouldn't be a package for it. **Don't be a dick about it.** And you're more than welcome to provide a package to make things easier for others, since you apparently were able to build it from source and run things perfectly fine.

**Comment by ikeydoherty** [comment]
docker can use various methods. For now, VirtualBox is low priority but at *some point* it will end up in the repo. Right now qemu is available. We'll let folks know when vbox is available.

**Comment by b1be** [comment]
Let's try to be civilised, and ask questions in a respectfull manner, for the offensive words I will post a script in "some" days on my mirror. Building from source is easy and safe, if you CAN READ the instructions.

**Comment by bet0x** [comment]
After reading this kinda of acting from SolusOS people i will not use it. This is how forking starts.

**Comment by JoshStrobl** [comment]
@b1be Any chance you could make the Virtualbox package instead of using a bash script. While I'm sure the bash script is safe and all, people have a tendency to trust something more if it is coming from the repo (where it can be vetted) as opposed to running a bash script that they are unlikely to read.

**Comment by ikeydoherty** [comment]
@bet0x then don't, but I invite you to re-read those comments.As much as folks may be using the wrong words right now, the reaction to b1be's comment had reason.

People are free to fork Solus as much as the like, however I truly emphasise **Good luck**. It's an enormous undertaking for anyone, a simple rebrand won't cut it. And I definitely won't tolerate a derived work acting in a virulent fashion.

In short, there seems to be a lot of overreacting on GitHub right now. No idea why. That's still no reason to believe we're in some form of fork crisis :]

> from SolusOS people
No, from community members. The only "Solus" person would be me. It's not SolusOS btw.

**Comment by ikeydoherty** [comment]
Closing, per https://plus.google.com/+IkeyDoherty/posts/SiHdw8XQPxh
